documentation/docs/20-core-concepts/20-load.md

```diff
@@ -417,6 +417,8 @@ export function load({ locals }) {
 
 > Make sure you're not catching the thrown redirect, which results in a noop.
 
+To navigate users from the client-side, you'll want to use [`goto`](modules#$app-navigation-goto) from [`$app.navigation`](modules#$app-navigation)
```

Comment:
You can (and should) use `throw redirect` inside a universal `load` function, so perhaps this would be more complete:

```suggestion
In the browser, you can also navigate programmatically outside of a `load` function using [`goto`](modules#$app-navigation-goto) from [`$app.navigation`](modules#$app-navigation).
```